podtato-server/kind/full_install.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+echo "Creating KIND cluster "
+
+kind create cluster --config=./cluster.yaml
+
+echo "Installing traefik"
+
+helm repo add traefik https://helm.traefik.io/traefik
+helm repo update
+helm install traefik traefik/traefik -f traefik_values.yaml
+
+echo "Exposing traefik dasboard"
+
+kubectl apply -f ./traefik_dashboard.yaml
```

Comment:
After this step perhaps perform the podtato head chart install?

https://github.com/cncf/podtato-head/tree/main/delivery/charts

e.g.
```
cd ../delivery/charts/podtatoserver
helm install podtato . 
```

Reply:
I was not sure which version people wanted to install so I wanted to give them the vanilla cluster setup